toolkit/go/pkg/log/log.go

```diff
@@ -53,10 +53,12 @@ func InitConfig(conf *pldconf.LogConfig) {
 	output := confutil.StringNotEmpty(conf.Output, *pldconf.LogDefaults.Output)
 	switch output {
 	case "file":
+		filename := confutil.StringNotEmpty(conf.File.Filename, *pldconf.LogDefaults.File.Filename)
+		rootLogger.Infof("Logs diverted to %s", filename)
```

Comment:
This confused me for a bit on the zeto tests (as they take a long time with no console output), so I thought this would be helpful to remind us where the logs are going.